doc/devel/index.rst

```diff
@@ -4,6 +4,15 @@
 Contribute
 ##########
 
+.. ifconfig:: releaselevel != 'dev'
+
+   .. important::
+
+      If you plan to contribute to Matplotlib, please read the
+      `development version <https://matplotlib.org/devdocs/devel/index.html>`_
+      of this document as it will have the most up to date installation
+      instructions, workflow process, and contributing guidelines.
+
```

Comment:
Where did this come from?  I didn't see it before and it doesn't show in the "force-push" diff :confused:

Reply:
I've had that happen for this PR a couple of times where changes on main get shown as diffs in this PR?  Not sure why, but a rebase on main fixes it